index.js

```diff
@@ -38,19 +38,15 @@ ManageBooks.prototype.delete = function (e) {
   this.display();
 };
```

Comment:
- [ ] I am getting an error when trying to add a book
![image](https://user-images.githubusercontent.com/62705947/206221823-5e63af5a-e357-42de-9700-ec61829a0ad4.png)
_It is comming from index.js on line 22_

- [ ] After fixing this make sure you have met other requirements as well getting the data from local storage

Reply:
@GzimAsani
The Local storage is working just fine.